### PR TITLE
Upgrade to node 4.1.1

### PR DESCRIPTION
**Pull request by aguspina**

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,39 @@
+0 info it worked if it ends with ok
```
Comment (flbulgarelli) [inline npm-debug.log:1]: Could you please remove this file?

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- '0.12'
+- '4.1.1'
```
Comment (flbulgarelli) [inline .travis.yml:3]: Is node 4.1.1 supported by Heroku?

**Comment by flbulgarelli** [comment]
This PR looks OK for me. But could you confirm we can actually deploy this node version to Heroku?

**Comment by aguspina** [comment]
In heroku tutorial forma config node apps... The screenshot they use is
using node 4.1.1 . So I inferred it is supported.

**Pull request merged**